packages/flutter_markdown/test/flutter_markdown_test.dart

```diff
@@ -94,6 +94,21 @@ void main() {
       expect(span.children[0].recognizer.runtimeType, equals(TapGestureRecognizer));
     });
   });
+
+  test("Changing config", () {
+    testWidgets((WidgetTester tester) {
+      tester.pumpWidget(new Markdown(data: "Data1"));
+      _expectTextStrings(_listElements(tester), <String>["Data1"]);
+
+      String stateBefore = WidgetFlutterBinding.instance.renderViewElement.toStringDeep();
+      tester.pumpWidget(new Markdown(data: "Data1"));
+      String stateAfter = WidgetFlutterBinding.instance.renderViewElement.toStringDeep();
+      expect(stateBefore, equals(stateAfter));
+
+      tester.pumpWidget(new Markdown(data: "Data2"));
+      _expectTextStrings(_listElements(tester), <String>["Data2"]);
+    });
+  });
```

Comment:
good test. I'd add some others that check that you also rebuild if only the style changes or only the syntax highlighter changes. (For those, you can do the toStringDeep trick to check that that changes, rather than using _expectTextStrings)